Makefile

```diff
@@ -45,7 +44,7 @@ endif
 MAKEFLAGS += --no-print-directory
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
-HOSTGO := env -u GOOS -u GOARCH -u GOARM -- go
+HOSTGO := env -u $(GOOS) -u $(GOARCH) -u $(GOARM) -- go
```

Comment:
This means to unset the values of GOOS, not GOOS itself. If you run `GOARCH=arm64 make`, it won't be unset. I don't believe this line should change now.

Reply:
Good catch, I've undid this change